relation.go

```diff
@@ -137,6 +137,11 @@ var relationDeserializationFuncs = map[int]relationDeserializationFunc{
 
 func newRelationImporter(v int, checker schema.Checker) func(map[string]interface{}) (*relation, error) {
 	return func(source map[string]interface{}) (*relation, error) {
+		// Some relations don't have status; remove any nil value
+		// so schema validation doesn't complain.
```

Comment:
Probably worth leaving a comment about potential import issues due to released bad code.

Reply:
done